src/main/webapp/i18n/de/exercise.json

```
@@ -241,6 +241,14 @@
                     "customMin": "Die Bonuspunkte müssen mindestens 0 betragen!",
                     "customMax": "Die Bonuspunkte dürfen höchstens 9999 betragen!"
                 },
+                "minTeamSize": {
+                    "min": "Die untere Schranke für die minimale Teamgröße ist eine Person.",
+                    "max": "Die obere Schranke für die minimale Teamgröße sind 99 Personen."
+                },
+                "maxTeamSize": {
+                    "min": "Die untere Schranke für die maximale Teamgröße ist eine Person.",
+                    "max": "Die obere Schranke für die maximale Teamgröße sind 99 Personen."
```

This phrasing might be less confusing (sorry for not coming up with that on monday). If you don't like it, ignore.
```suggestion
                    "min": "Die minimale Teamgröße darf nicht weniger als eine Person betragen.",
                    "max": "Die minimale Teamgröße darf nicht mehr als 99 Personen betragen."
                },
                "maxTeamSize": {
                    "min": "Die maximale Teamgröße darf nicht weniger als eine Person betragen.",
                    "max": "Die maximale Teamgröße darf nicht mehr als 99 Personen betragen."
```